<?xml version="1.0"?>
<!--
  Copyright (C) 2018-2021 LEIDOS.

  Licensed under the Apache License, Version 2.0 (the "License"); you may not
  use this file except in compliance with the License. You may obtain a copy of
  the License at

  http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  License for the specific language governing permissions and limitations under
  the License.
-->

<!-- 
Primary launch file for TrajectoryExecutor node
Loads parameters and configures logging for node, defaults to screen output.
 -->
<launch>
    <!-- Trajectory Executor Node -->
    <node pkg="trajectory_executor" type="trajectory_executor_node" name="trajectory_executor_node">
        <rosparam command="load" file="$(find trajectory_executor)/config/trajectory_executor.yaml" />
    </node>
</launch>